src/components/HelpTopics/EnableHelpTopicContext.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { HelpTopicContext } from '@patternfly/quickstarts';
+import { useChromeHelpTopicsApi } from './help-topics';
+
+type EnableHelpTopicContextType = {
+  addTopicName: (topicName: string) => void;
+  deleteTopicName: (topicName: string) => void;
+};
+
+const EnableHelpTopicContext = React.createContext<EnableHelpTopicContextType>({
+  addTopicName: () => {},
+  deleteTopicName: () => {},
+});
+
+export const EnableHelpTopicContextProvider: React.FC = ({ children }) => {
+  const [helpTopics, setHelpTopics] = React.useState<string[]>([]);
```

Comment:
Why do you need to maintain a separate context of enabled help topics when chrome does it already at root level? Shouldn't that be used as a direct API?